.devcontainer/devcontainer.json

```diff
@@ -61,6 +61,11 @@
                 "ms-python.python",
                 "ms-python.black-formatter",
                 "github.vscode-github-actions",
+                "actboy168.tasks",
+                "charliermarsh.ruff",
+                "ms-python.mypy-type-checker",
+                "eamodio.gitlens",
+                "GitHub.vscode-pull-request-github",
                 "ms-azuretools.vscode-docker"
```

Comment:
Did you want veryone on the team to have all these plugins? Might be worth a chat - imaging at least some are valuable to all? I have found gitlens slow in the past so have previously avoided it but that may be old news.

Reply:
I've only left the 2 that I think are useful for everyone. I've tried looking into how to have a personal set of extensions for devcontainers, but couldn't find anything useful. Will make do with re-installing the extensions whenever the container gets rebuilt.